test/jdk/javax/management/remote/mandatory/notif/DeadListenerTest.java

```diff
@@ -75,11 +75,11 @@ public static void main(String[] args) throws Exception {
         RMIConnectorServer cs = new RMIConnectorServer(url, null, rmiServer, mbs);
         cs.start();
         JMXServiceURL addr = cs.getAddress();
-        assertTrue("No connections in new connector server", rmiServer.connections.isEmpty());
+        assertTrue("Expected no connections in new connector server", rmiServer.connections.isEmpty());
```

Comment:
Changes to this file seem unrelated to the removal of the feature - could you confirm that they were intended (no issue if they were intended).

Reply:
Yes, they were intentional, should have made that clear.  I had a failure from this test at some point, and the assert message made no sense to me.
e.g. "New value of count1 == 1" is shown, but that's what was expected and not what was seen.